aspnet/fundamentals/localization/sample/Controllers/AccountController.cs

```diff
@@ -0,0 +1,467 @@
+﻿using System;
```

Comment:
@ryanbrandenburg That's generated by the templates and is needed to build the project. Once the CLI tooling works I'll regenerate the project and it will include the account controller.